templates/CRM/Core/SortableCheckboxes.tpl

```diff
@@ -0,0 +1,20 @@
+<ul class="crm-checkbox-list{if $sortable} crm-sortable{/if}">
```

Comment:
I think it worth writing in a comment in this template file what variables this template expect .